### ISSUE TITLE
Actively sync users to the forum

### ISSUE THREAD

**Issue by raboof** [CoderDojo/community-platform#939]
Currently, users can use the forums to send each other (private) messages.

This only works for users that already have used their forum account, as this is not automatically created when you sign up.

It would be nice to 'actively' create forum accounts for all users, so even though I haven't used to forum yet, I'm reachable there for others.

**Comment by Wardormeur** [comment]
I totally agree :+1: 
However it probably needs a review for everything related to children protection, as I don't think opening an account for an U13 is necessarly a good thing

**Comment by raboof** [comment]
Makes sense - perhaps we should first do it for champions/mentors, and perhaps later consider what to do with other user types?

**Comment by rosalanghammer** [comment]
1 to 1 messaging for anyone under 18 is not available and isn't planned for the forums or elsewhere. Ideally opening this up for (potential) Champions & mentors would be a great first step.
